### ISSUE TITLE
Function suggestion: menu and built-in script of script node

### ISSUE THREAD

**Issue by vkter** [nortikin/sverchok#4843]
1. The search menu should be replaced by the menu directory, similar to the example
![image](https://user-images.githubusercontent.com/74725748/209489448-3c8a5bef-356e-4aa7-93e1-50bc540eca49.png)
2 Attach a custom path to allow users to add their own script directory on the basis of the built-in directory
3 Collect the current sverchok open source script projects and add them to sverchok, which will facilitate users

**Comment by Durman** [comment]
The search property is needed to attach scripts from Blender text editor and can't be replaced with menu of existing scripts. The menu, probably, can exist as independent element.

**Comment by vkter** [comment]
Of course, the additional menu directory is better. Can developers familiar with this node consider this suggestion?
